libs/domains/service-terraform/feature/src/lib/terraform-variables-settings/terraform-configuration-settings/terraform-configuration-settings.tsx

```diff
@@ -156,37 +168,80 @@ export const TerraformConfigurationSettings = ({
+                  className="flex flex-col gap-5"
+                  defaultValue={'kubernetes' in field.value ? 'kubernetes' : 'user_provided'}
+                  onValueChange={(value) =>
+                    field.onChange(value === 'kubernetes' ? { kubernetes: {} } : { user_provided: {} })
+                  }
+                >
+                  <label className="grid grid-cols-[16px_1fr] gap-3">
+                    <RadioGroup.Item value="kubernetes" />
+                    <div className="flex flex-col gap-1">
+                      <span className="font-medium">Use backend generated by Qovery (Kubernetes)</span>
+                      <ul className="list-disc pl-3 text-neutral-350">
+                        <li>
+                          Not recommended for production environments (as there won't be any backup of the state).
+                        </li>
+                        <li>Qovery manages the Terraform state in your cluster.</li>
+                        <li>We override backend.tf at runtime; you don't need to configure a backend.</li>
+                      </ul>
+                    </div>
+                  </label>
+                  <label className="grid grid-cols-[16px_1fr] gap-3">
+                    <RadioGroup.Item value="user_provided" />
+                    <div className="flex flex-col gap-1">
+                      <span className="font-medium">Use your own backend</span>
+                      <ul className="list-disc pl-3 text-neutral-350">
+                        <li>Qovery does not override your files.</li>
+                        <li>You must configure your backend and set the required environment variables.</li>
+                        <li>Credentials are provided via environment variables.</li>
+                      </ul>
+                    </div>
+                  </label>
+                </RadioGroup.Root>
+              )}
             />
-          )}
-        />
+
+            {'kubernetes' in backend && (
+              <div className="mt-4 flex w-full flex-col gap-2 rounded border border-neutral-250 px-4 py-3 text-sm">
+                <span className="font-medium">Access Terraform state from your local machine</span>
+                <p className="text-neutral-350">
+                  {isSettings ? (
+                    <span>
+                      You can configure and access the state of your Terraform service from your local machine with the
+                      following CLI command.
+                    </span>
+                  ) : (
+                    <span>
+                      Once your service is created, you can configure and access the state of your Terraform service
+                      from your local machine with the following CLI command.
+                    </span>
+                  )}
+                </p>
+                <div className="flex items-center justify-between gap-6 rounded-sm bg-neutral-150 p-3 text-neutral-400">
```

Comment:
Add an alignement in the top like what we have with "Access info" modal

```suggestion
                <div className="flex justify-between gap-6 rounded-sm bg-neutral-150 p-3 text-neutral-400">
```